festim/exports/txt_export.py

```diff
@@ -46,22 +45,61 @@ def when_is_next_time(self, current_time):
                 return time
         return None
 
-    def write(self, current_time, steady):
+    def is_it_first_time_to_export(self, current_time, nb_iteration):
```

Comment:
I don't think we need a separate method for that. This requires that we pass another argument to the write method.

Instead we could have a `._first_time` attribute as a boolean, initialised as `True` to simplify things.

I'll make a PR to your branch to show what I'm talking about

Reply:
> Instead we could have a ._first_time attribute as a boolean, initialised as True to simplify things.

Got it. It surely makes things better!

Reply:
See [this PR](https://github.com/KulaginVladimir/FESTIM/pull/1) to your fork